bait-files/src/main/webapp/WEB-INF/Error404.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```

Comment:
Under what circumstances does this page manifest?

Reply:
Removed